README.md

```diff
@@ -43,10 +43,11 @@ You can tell logspout to ignore specific containers by setting an environment va
 
 Using the [httpstream module](http://github.com/gliderlabs/logspout/blob/master/httpstream), you can connect with curl to see your local aggregated logs in realtime. You can do this without setting up a route URI.
 
-	$ docker run -d -p 8000:8000 \
-		-v=/var/run/docker.sock:/var/run/docker.sock \
-		progrium/logspout
-	$ curl $(docker port `docker ps -lq` 8000)/logs
```

Comment:
If we do something like `--publish=127.0.0.1:8000:8000` (to avoid having the port be externally available), then we can just hard-code 127.0.0.1 as before rather than look it up using ```docker port `docker ps -lq` 8000```.